### PR TITLE
Organize racingj and thrilld inputs up.

### PR DESCRIPTION
**Pull request by Hydreigon223**
Take two of a pull request I did about a week ago per hap's suggestion to assign the clutch pedal to a different player. I can now add the other versions of Racing Jam that have a handbrake and/or clutch installed.

New clones marked not working:
Racing Jam (GM676 Ver UAC/EAC/JAC/AAC)
Racing Jam (GN676 Ver UAC/EAC/JAC/AAC)
Racing Jam: Chapter II (GM888 Ver UAC/EAC/JAC/AAC)
Racing Jam: Chapter II (GN888 Ver UAC/EAC/JAC/AAC)

### CODE REVIEW

**Comment by Hydreigon223** [comment]
Going to wait for everyone to agree with these changes before I email the timekeeper files for these added clones.

**Comment by 987123879113** [comment]
LGTM. I previously left some notes about the remaining different cabinet types that still needed to be implemented + tested so that someone who had more interest could fill out the rest of the cabinet variations controls later, so this kind of PR is exactly what I was hoping would happen eventually. Thank you @Hydreigon223.

**Comment by Hydreigon223** [comment]
With this coming to an agreement, I will gladly email the timekeeper files soon.

File: src/mame/konami/nwk-tr.cpp [hunk 1]
```diff
@@ -654,12 +607,41 @@ static INPUT_PORTS_START( nwktr )
 	PORT_BIT( 0xfff, 0x000, IPT_PEDAL2 ) PORT_NAME("Brake Pedal") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60)
 
 	PORT_START("ANALOG4")
-	PORT_BIT( 0xfff, 0x000, IPT_AD_STICK_Y ) PORT_NAME("Handbrake Lever") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60) PORT_REVERSE
+	PORT_BIT( 0xfff, 0x000, IPT_CUSTOM )
 
 	PORT_START("ANALOG5")
-	PORT_BIT( 0xfff, 0x000, IPT_PEDAL3 ) PORT_NAME("Clutch Pedal") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60)
+	PORT_BIT( 0xfff, 0x000, IPT_CUSTOM )
```
Comment (cuavas) [inline src/mame/konami/nwk-tr.cpp:613]: Please replace the `IPT_CUSTOM` placeholders with `IPT_UNKNOWN`.  You should only use `IPT_CUSTOM` with custom handler callbacks.

**Review by cuavas — COMMENTED** [review]
Please update the list of added systems in your PR description to match the actual

**Pull request merged**